README.md

```diff
@@ -1,140 +1,127 @@
+- [Who/what is this for?](#whowhat-is-this-for)
+- [Getting started](#getting-started)
+  - [Requirements](#requirements)
+- [Changes to make post-creation](#changes-to-make-post-creation)
+- [Changes to consider post-creation](#changes-to-consider-post-creation)
+- [Licence](#licence)
+- [Contributing](#contributing)
+- [Acknowledgements](#acknowledgements)
 
-You need to have
-[cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/installation.html#install-cookiecutter)
-installed.
+## Who/what is this for?
 
-This project runs on Python 3.5+. To install required Python packages via `pip`, first [set up a Python virtual
-environment](#creating-a-python-virtual-environment); this ensures you do not install the packages globally. Note there
-are additional [requirements](./%7B%7B%20cookiecutter.repo_name%20%7D%7D/README.md#requirements) for the output.
+This template is for any HM Government analyst(s) who codes! It's main purpose is to:
 
-To create a new repository structure from this cookiecutter, install the required Python packages via `pip` by running
-the following command:
+1. Provide a lightweight, Agile-like approach to Analytical Quality Assurance (AQA)
+2. Help quickly set up standardised project structures.
 
-```shell script
-make requirements
-```
-
-To make developments to this project, install the development Python packages via `pip` using:
-
-```shell script
-make requirements-dev
-```
-
-Once you have installed the packages, remember to [set up pre-commit hooks](#installing-pre-commit-hooks).
-
-### Creating a Python virtual environment
+For more information about AQA, see [The Aqua Book][aqua-book], and its [resources][aqua-book-resources].
 
-Creating a Python virtual environment depends on whether you are using [base Python](#base-python-interpreter) or
-[Anaconda](#anaconda-interpreter) as your interpreter.
+Rather than using different files and locations to store AQA and documentation, our intention is to centralise as much
+of this as possible onto your Git repository hosting platform, e.g. GitHub or GitLab.
 
-#### Base Python interpreter
+We use nudges, such as checklists in pull/merge requests, to minimise the burden on contributors and reviewers to
+complete AQA checks. This results in faster iterative development and deployment, whilst ensuring HM Government-wide
+standards on assurance are met.
 
-If you are using base Python, there are multiple ways to create virtual environments in Python using `pip`, including
-(but not limited to):
+We have also included [pre-commit hooks][pre-commit] to prevent accidental committing of secrets, large data files, and
+Jupyter notebook outputs for security purposes.
 
-- [`venv`](https://docs.python.org/3/tutorial/venv.html);
-- [`virtualenv`](https://virtualenv.pypa.io/en/stable/);
-- [`pipenv`](https://github.com/pypa/pipenv); and
-- [`pyenv`](https://github.com/pyenv/pyenv) with its `virtualenv` [plugin](https://github.com/pyenv/pyenv-virtualenv).
+## Getting started
```

Comment:
```suggestion
## To start using govcookiecutter for your projects
```

Here as below, I'd be more explicit so that it is clearer what is needed if you just want to use the template vs. you want to contribute to the ukgovdatascience cookiecutter project

Reply:
Resolved in commit [`79b15e3`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/79b15e386e5c4561c6d8393f52edf25568d1c366).